initrd/bin/config-gui.sh

```diff
@@ -65,6 +65,11 @@ while true; do
         'N' " $(get_config_display_action "$CONFIG_AUTOMATIC_POWERON") automatic power-on"
     )
 
+    # Debugging option always available
+    dynamic_config_options+=(
+        'Z' " $(get_config_display_action "$CONFIG_DEBUG_OUTPUT") $CONFIG_BRAND_NAME Debug and function tracing mode"
```

Comment:
What do you think about `"(...) $CONFIG_BRAND_NAME debug and function tracing`?

Don't think 'mode' adds much so I'd trim it.  Lower-cased debug, our capitalization is a bit all over the place but _mostly_ I think they capitalize Cool Feature Names (like "Firmware Blob Jail", "Platform Locking", "Restricted Boot"), this feels more like a matter-of-fact thing like "automatic power-on" / "USB automatic boot".

Reply:
Agreed on mode -> output